flink-optimizer/src/test/java/org/apache/flink/optimizer/custompartition/GroupingPojoTranslationTest.java

```diff
@@ -176,19 +177,19 @@ public void testCustomPartitioningTupleInvalidType() {
 			fail(e.getMessage());
 		}
 	}
-	
+
 	@Test
 	public void testCustomPartitioningTupleInvalidTypeSorted() {
 		try {
 			ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
-			
+
 			DataSet<Pojo3> data = env.fromElements(new Pojo3())
 					.rebalance().setParallelism(4);
-			
```

Comment:
Many whitespace reformattings here. If you use IntelliJ, please set it to not auto-trim whitespaces. It blows up the diffs and distracts from actual changes.